driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideOperationsTimeoutProseTest.java

```diff
@@ -704,6 +707,40 @@ public void test10CustomTestWithTransactionUsesASingleTimeoutWithLock() {
         }
     }
 
+    /**
+     * Not a prose spec test. However, it is additional test case for better coverage.
+     */
+    @Tag("setsFailPoint")
+    @Test
+    @DisplayName("Should ignore wTimeoutMS of WriteConcern to initial and subsequent commitTransaction operations")
+    @Disabled //TODO JAVA-5425
```

Comment:
An observation from [JAVA-5425](https://jira.mongodb.org/browse/JAVA-5425): TimeoutContext is null in subsequent commitTransaction operations.

Reply:
Good catch